lib/ohai/application.rb

```diff
@@ -56,6 +56,12 @@ class Ohai::Application
     description: "Set the log file location, defaults to STDOUT - recommended for daemonizing",
     proc: nil
 
+  option :target,
+    short: "-t TARGET",
+    long: "--target TARGET",
+    description: "Target a remote location to invoke Ohai",
```

Comment:
```suggestion
    description: "A remote target to run Ohai against",
```

Reply:
that code is all "TBD" placeholder to just bootstrap it, i'm pretty sure it changes and i'm probably copying chef-client's api here and then adding the URI feature to the chef-client code.